ajax/libs/gijgo/1.3.0/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "gijgo",
+  "version": "1.3.0",
+  "description": "Gijgo is a set of free open source javascript controls distributed under MIT License. All widgets are high performance, built on top of the jQuery JavaScript Library with built-in support for Bootstrap, Material Design and Font Awesome. They are designed to saves you time and scales with your development process.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/atatanasov/gijgo.git"
+  },
+  "homepage": "http://gijgo.com/",
+  "filename": "js/gijgo.min.js",
+  "keywords": [
+    "jquery-plugin",
+    "jquery",
+    "gijgo",
+    "javascript",
+    "plugin",
+    "grid",
+    "datagrid",
+    "table",
+    "dialog",
+    "modal",
+    "draggable",
+    "droppable",
+    "tree",
+    "treeview"
+  ],

```

Comment:
The file structure should look like
```
ajax
└── libs
    └── gijgo
        ├── 1.3.0
        └── package.json
``` 

Please remove `js` and `css` folder under `ajax/libs/gijgo`. Thanks.